community_tasks/arabic_evals.py

```diff
@@ -61,7 +63,8 @@ def __init__(
             hf_subset=hf_subset,
             prompt_function="mmlu_arabic",
             hf_repo="OALL/Arabic_MMLU",
-            metric=["loglikelihood_acc"],
+            metric=["loglikelihood_acc_norm"],
+            # metric=["loglikelihood_acc"],
```

Comment:
```suggestion
```

Reply:
For the future, it's important to not change tasks once you have submitted them :)